CMakeLists.txt

```diff
@@ -176,7 +176,11 @@ if (Boost_FOUND)
 endif ()
 # We have to import LLVM's cmake definitions to build sea-dsa
 # Borrowed from sea-dsa's CMakeLists.txt
-find_package (LLVM ${LLVM_SHORT_VERSION} CONFIG)
+# Borrowed from https://github.com/banach-space/llvm-tutor/commit/72cb20d058b9b3f51717c7a17607f7a4c7398642
+find_package (LLVM 11.1 REQUIRED CONFIG)
```

Comment:
Does this have to be hard-coded now? That kind of sucks.

Reply:
Maybe we can add minor version to it and specify 11.1.0 as the full llvm version in bin/versions.

Reply:
I don't get it. Currently we have 3 versions floating around: 11, 11.0.1, and this weird 11.1. What is your suggestion exactly?

Reply:
We take the first 2 segments instead of 1 from `LLVM_FULL_VERSION` here. So if `LLVM_FULL_VERSION` is 11.1.0, we use `11.1` here whereas if it's 10.0.1, we use 10.0.

Reply:
I just pushed a new version. Could you take a look at it?